media_driver/linux/common/vp/ddi/media_libva_vp.c

```diff
@@ -1159,7 +1159,7 @@ DdiVp_SetProcPipelineParams(
         // set background colorfill option
         pVpHalRenderParams->pColorFillParams->Color     = pPipelineParam->output_background_color;
         pVpHalRenderParams->pColorFillParams->bYCbCr    = false;
-        pVpHalRenderParams->pColorFillParams->CSpace    = CSpace_sRGB;
+        pVpHalRenderParams->pColorFillParams->CSpace    = pVpHalSrcSurf->ColorSpace;
```

Comment:
Can we use some default value in case app does not set color space as the below? 

pVpHalRenderParams->pColorFillParams->CSpace = pVpHalSrcSurf->ColorSpace ? pVpHalSrcSurf->ColorSpace : CSpace_sRGB;

Reply:
Thank you for the reminder, I have amended this.